.github/workflows/go.yml

```
@@ -0,0 +1,49 @@
+name: Go
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+    - "main"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ "1.19", "1.20" ]
```

```suggestion
        go: "1.20.x"
```

We can just test it with the latest, no need for last-1 version

---

fixed